cf/test/run_tests.py

```diff
@@ -42,6 +44,13 @@ def randomise_test_order(*_args):
 shuffle(all_test_cases._tests)
 testsuite.addTests(all_test_cases)
 
+# Add a test suite for doctests
+# https://docs.python.org/3/library/doctest.html#unittest-api
+testsuite_doctests = unittest.TestSuite()
+for importer, name, ispkg in \
+        pkgutil.walk_packages(cf.__path__, cf.__name__ + '.'):
```

Comment:
It is more in line with PEP8 to avoid backslashes for line continuation, so as one minor comment I would suggest changing this as otherwise it might break our test running pycodestyle:

```suggestion
for importer, name, ispkg in (
        pkgutil.walk_packages(cf.__path__, cf.__name__ + '.')):
```